plugins/search-backend-module-pg/src/PgSearchEngine/PgSearchEngine.ts

```diff
@@ -116,9 +118,42 @@ export class PgSearchEngine implements SearchEngine {
     this.highlightOptions = highlightOptions;
     this.indexerBatchSize =
       config.getOptionalNumber('search.pg.indexerBatchSize') ?? 1000;
+
+    this.normalization = this.getNormalizationValue(config);
     this.logger = logger;
   }
 
+  private getNormalizationValue(config: Config) {
+    const normalizationConfig =
+      config.getOptional('search.pg.normalization') ?? 0;
```

Comment:
This new config should also be added here: https://github.com/backstage/backstage/blob/master/plugins/search-backend-module-pg/config.d.ts